docs/guide/installation.rst

```diff
@@ -0,0 +1,58 @@
+.. _guide_installing:
+
+************
+Installation
+************
+
+The SunPy project `maintains a range of packages <https://sunpy.org/project/affiliated>`__ that leverage the wider ecosystem of scientific Python packages for solar physics.
+
+conda
+=====
+Full instructions for installing using conda are in :ref:`installing`.
+This is the recommended way to install sunpy, because it creates a fresh Python environment that is independent from any other Python install or packages on your system.
```

Comment:
```suggestion
This is the recommended way to install ``sunpy``, because it creates a fresh Python environment that is independent from any other Python install or packages on your system.
```